src/lib/components/ProductForm.svelte

```diff
@@ -629,20 +664,24 @@
 				type="checkbox"
 				bind:checked={product.hasVariations}
 				name="hasVariations"
-				disabled={!product.standalone}
 			/>
 			Product has light variations (no stock difference)
 		</label>
 		{#if product.hasVariations}
 			{#each [...(product.variations || []), ...Array(variationLines).fill( { name: '', value: '', price: 0 } )].slice(0, variationLines) as variation, i}
```

Comment:
I know this was not introduced in this commit, but please clean this up... You can bind this expression to a symbol explaining right bellow the `{#if product.hasVariations}` explaining what it is.